finagle-core/src/main/scala/com/twitter/finagle/Filter.scala

```diff
@@ -48,28 +50,11 @@ abstract class Filter[-ReqIn, +RepOut, +ReqOut, -RepIn]
    *
    * @param next another filter to follow after this one
    */
-  def andThen[Req2, Rep2](next: Filter[ReqOut, RepIn, Req2, Rep2]) =
-    new Filter[ReqIn, RepOut, Req2, Rep2] {
-      def apply(request: ReqIn, service: Service[Req2, Rep2]) = {
-        val svc: Service[ReqOut, RepIn] = new Service[ReqOut, RepIn] with Proxy {
-          // note that while `Service.rescue` could be used here it would
-          // entail an extra allocation.
-          def apply(request: ReqOut): Future[RepIn] = {
-            try {
-              next(request, service)
-            } catch {
-              case NonFatal(e) => Future.exception(e)
```

Comment:
It seems like we've dropped this behavior? Do we need to lift exceptions in `build` to Futures?